src/data/html-quiz.ts

```diff
@@ -1,4 +1,16 @@
 const htmlQuiz = [
+  {
+    Question:
+    "Which of the following is the correct way of creating an hyperlink in HTML?",
+    Answer: "<a href= “www.freecodecamp.org”> freeCodeCamp </a>",
+    Distractor1: "<a>www.freecodecamp.org <freeCodeCamp /a>",
+    Distractor2: "<a href=“ www.freecodecamp.org” freeCodeCamp /a>",
+    Distractor3: "<a link=“ www.freecodecamp.org” freeCodeCamp > </a>",
```

Comment:
let's just go with single quotes here

```suggestion
    Answer: "<a href= 'www.freecodecamp.org'> freeCodeCamp </a>",
    Distractor1: "<a>www.freecodecamp.org <freeCodeCamp /a>",
    Distractor2: "<a href='www.freecodecamp.org' freeCodeCamp /a>",
    Distractor3: "<a link='www.freecodecamp.org' freeCodeCamp > </a>",
```